James Bierly Javascript Challenge

jbierly1 opened the pull request:
@juliankohlman

juliankohlman commented:
## Great

- Overall: You did well James, couple areas you can improve your overall code, but you've definitely shown you understand the material. Keep it up, and let me know if you have questions about your feedback.
- awesome use of the regex on line # 151
- Functions good explanation on the nestedFunction() question.

## Requested Improvements

- Answers.md Re-read [.forEach()](https://devdocs.io/javascript/global_objects/array/foreach)
- Answers.md: on the whole your answers are good, but I'd look over those topics for question #1, and #3, just to solidify your knowledge.
- arrays-objects: line #133 check your syntax when using `Array.sort()` it's causing a `TypeError`.
- line # 142 & 127 why are you starting i at 1? What about poor Cynde at graduates[0]?
- Functions: I'd use a more descriptive variable name for `answ` maybe `result` or `answer`.
- There's some uneven indentation in some of the files. If you're using a code-formatter you can tweak the indentation settings to fix that.
- Classes: On lines #8, #13 what is the `anw` variable representing in your CuboidMaker object?

## Rating: {1-3} => 
- (2.5) can easily be a 3 with fix to `arrays-objects.js`

jbierly1 commented:
ok, I reworded 1 and added to 3 on the Answers.md.  arrays-objects #133 I'd
fixed in Repl.it but forgot to copy back into VSCode.  Should work now.
142 and 127 are fixed now.  Cynde can graduate I guess. I spelled out
answer in Functions.  In terms of the indentation, I assume we'll talk
about that on Saturday.  I was using Prettier for this code, so you
probably know how to adjust it properly.  In classes anw was just my name
for the variable I was going to return... I went ahead and rewrote it so
it's "answer" as well.  Thanks for the detailed feedback, talk to you soon!


On Tue, Aug 28, 2018 at 11:29 AM Julian Jay Kohlman <
notifications@github.com> wrote:

> Great
>
>    - Overall: You did well James, couple areas you can improve your
>    overall code, but you've definitely shown you understand the material. Keep
>    it up, and let me know if you have questions about your feedback.
>    - awesome use of the regex on line # 151
>    - Functions good explanation on the nestedFunction() question.
>
> Requested Improvements
>
>    - Answers.md Re-read .forEach()
>    <https://devdocs.io/javascript/global_objects/array/foreach>
>    - Answers.md: on the whole your answers are good, but I'd look over
>    those topics for question #1
>    <https://github.com/LambdaSchool/Sprint-Challenge--JavaScript/pull/1>,
>    and #3
>    <https://github.com/LambdaSchool/Sprint-Challenge--JavaScript/pull/3>,
>    just to solidify your knowledge.
>    - arrays-objects: line #133
>    <https://github.com/LambdaSchool/Sprint-Challenge--JavaScript/pull/133>
>    check your syntax when using Array.sort() it's causing a TypeError.
>    - line # 142 & 127 why are you starting i at 1? What about poor Cynde
>    at graduates[0]?
>    - Functions: I'd use a more descriptive variable name for answ maybe
>    result or answer.
>    - There's some uneven indentation in some of the files. If you're
>    using a code-formatter you can tweak the indentation settings to fix that.
>    - Classes: On lines #8
>    <https://github.com/LambdaSchool/Sprint-Challenge--JavaScript/pull/8>,
>    #13
>    <https://github.com/LambdaSchool/Sprint-Challenge--JavaScript/pull/13>
>    what is the anw variable representing in your CuboidMaker object?
>
> Rating: {1-3} =>
>
>    - (2.5) can easily be a 3 with fix to arrays-objects.js
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/LambdaSchool/Sprint-Challenge--JavaScript/pull/560#issuecomment-416631327>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/Ac3L5PWAWUTjvTs1lRqMfebk-9tXTxoKks5uVWHugaJpZM4WMcc1>
> .
>
